src/test/java/uk/gov/companieshouse/api/accounts/service/impl/ApprovalServiceTest.java

```diff
@@ -102,6 +109,7 @@ public void createApprovalDuplicateKey() throws DataException {
     void createApprovalMongoExceptionFailure() throws DataException {
         doReturn(approvalEntity).when(approvalTransformer).transform(ArgumentMatchers
             .any(Approval.class));
+        doReturn(new Errors()).when(approvalValidator).validateApproval(approval, request);
         when(approvalRepository.insert(approvalEntity)).thenThrow(mongoException);
         Executable executable = () -> {
```

Comment:
i know this isn't in this PR but we could change bit of code to be one line?
FROM:
```java
Executable executable = () -> {
            approvalService.create(approval, transaction, "", request);
        };
        assertThrows(DataException.class, executable);
```

TO:
```java
        assertThrows(DataException.class, () -> approvalService.create(approval, transaction, "", request));
```

Reply:
The file is formatted to Companieshouse Java coding standards, I would be reluctant to change the code mentioned as I don't think a minor increase in readability warrants breaking the coding standards for. I could probably use a smaller naming convention for params to fit it on one line if you are really keen on this idea?

Reply:
It's not really a code standards issue, its more of a code efficiency issue, we are creating a redundant executable and then using it in the assert. Instead we can remove the assignment to an executable and just use the lambda in the assert. This saves the redundant assignment to the executable variable.

Reply:
My mistake that change is made.